crates/bevy_sprite/src/render/mod.rs

```diff
@@ -469,9 +469,10 @@ const QUAD_UVS: [Vec2; 4] = [
     Vec2::new(0., 0.),
 ];
 
-#[derive(Component, Eq, PartialEq, Copy, Clone)]
+#[derive(Component)]
 pub struct SpriteBatch {
-    image_handle_id: HandleId,
+    range: Range<u32>,
+    image: Handle<Image>,
```

Comment:
Is there any noticeable performance impact of using a full `Handle<Image>` instead of `HandleId`?

Reply:
I don't think there would be since it's only one per batch and the size difference is small, but I also don't recall why I made this change. I believe I noticed that in UiBatch we were using a handle instead of an id, tried swapping to an id for consistency and then swapped back for some reason that I now can't remember. Ideally since HandleId is smaller the extracted ui nodes should be using ids as well. I will try and swap them all to ids and see if I rediscover why I made this change.

Reply:
Ok so the comment attached to the image_handle_id field on ExtractedSprite says:
```
PERF: storing a `HandleId` instead of `Handle<Image>` enables some optimizations (`ExtractedSprite` becomes `Copy` and doesn't need to be dropped)
```
We don't need ExtractedSprite to be Copy anymore so the comment here no longer applies, but I have swapped back to ids since they should be smaller